Kernel/syscall_net.c

```diff
@@ -136,7 +136,7 @@ arg_t _netcall(void)
 {
 	uint8_t flags = 0;
 	uint8_t cn;
-	arg_t *ap = udata.u_net.args;
```

Comment:
this assignment was overwritten later. can you imagine the amount of debugging required to find it?!